recipe/meta.yaml

```
@@ -53,6 +53,12 @@ outputs:
         - scipy
         - semantic_version >=2.10
         - toml
```

btw it seems we can also add this to be safe?
```suggestion
        - semantic_version >=2.10
        - toml
      run_constrained:
        - pennylane >=0.29.0
```
This is saying: If `pennylane` is not installed to an environment where `pennylane-core` is installed, it's fine. But if it's installed, it must not clobber with `pennylane-core`, and hence it must be of a recent version after which the splitting (new format) was introduced.

---

Make sense, thanks.